metricbeat/module/apache/status/data.go

```diff
@@ -211,40 +211,50 @@ func eventMapping(body io.ReadCloser, hostname string, metricset string) common.
 	}
 
 	event := common.MapStr{
-		"totalAccesses":           totalAccesses,
-		"totalKBytes":             totalKBytes,
-		"uptime":                  uptime,
-		"cpuLoad":                 cpuLoad,
-		"cpuUser":                 cpuUser,
-		"cpuSystem":               cpuSystem,
-		"cpuChildrenUser":         cpuChildrenUser,
-		"cpuChildrenSystem":       cpuChildrenSystem,
-		"reqPerSec":               reqPerSec,
-		"bytesPerSec":             bytesPerSec,
-		"bytesPerReq":             bytesPerReq,
-		"busyWorkers":             busyWorkers,
-		"idleWorkers":             idleWorkers,
-		"connsTotal":              connsTotal,
-		"connsAsyncWriting":       connsAsyncWriting,
-		"connsAsyncKeepAlive":     connsAsyncKeepAlive,
-		"connsAsyncClosing":       connsAsyncClosing,
-		"serverUptimeSeconds":     serverUptimeSeconds,
-		"load1":                   load1,
-		"load5":                   load5,
-		"load15":                  load15,
-		"scbStartingUp":           totalS,
-		"scbReadingRequest":       totalR,
-		"scbSendingReply":         totalW,
-		"scbKeepalive":            totalK,
-		"scbDnsLookup":            totalD,
-		"scbClosingConnection":    totalC,
-		"scbLogging":              totalL,
-		"scbGracefullyFinishing":  totalG,
-		"scbIdleCleanup":          totalI,
-		"scbOpenSlot":             totalDot,
-		"scbWaitingForConnection": totalUnderscore,
-		"scbTotal":                totalAll,
-		"hostname":                hostname,
+		"hostname":      hostname,
+		"totalAccesses": totalAccesses,
+		"totalKBytes":   totalKBytes,
+		"reqPerSec":     reqPerSec,
+		"bytesPerSec":   bytesPerSec,
+		"bytesPerReq":   bytesPerReq,
+		"busyWorkers":   busyWorkers,
+		"idleWorkers":   idleWorkers,
+		"uptime": common.MapStr{
+			"serverUptimeSeconds": serverUptimeSeconds,
```

Comment:
what is the difference between these to varialbes (didn't check the docs). Do we need both? Could one be calculated?